sample-queries/sample-queries.json

```diff
@@ -1965,8 +1965,8 @@
         "method": "GET",
         "humanName": "get ediscovery case",
         "requestUrl": "/beta/compliance/ediscovery/cases/{caseId}",
-        "docLink": "https://docs.microsoft.com/en-us/graph/api/ediscoverycase-get?view=graph-rest-beta",
-        "tip": "This query requires a case id. To find the id of a case; you can run: GET https://graph.microsoft.com/beta/compliance/ediscovery/cases",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/ediscovery-ediscoverycase-get?view=graph-rest-beta",
+        "tip": "Go to Modify Permissions tab and consent to eDiscovery.Read.All or eDiscovery.ReadWrite.All permission to run this query",
```

Comment:
It's still important to add this sentence that was there before: This query requires a case id. To find the id of a case; you can run: GET https://graph.microsoft.com/beta/compliance/ediscovery/cases to help users know how to get the id.

Reply:
Fixed.